src/sonic-yang-models/yang-models/sonic-vlan.yang

```diff
@@ -178,7 +182,7 @@ module sonic-vlan {
 
 				leaf name {
 					type string {
-						pattern 'Vlan([0-9]{1,3}|[1-3][0-9]{3}|[4][0][0-8][0-9]|[4][0][9][0-4])';
```

Comment:
The original regex only allows vlan1-4094, so this change should not be necessary. But there are other discrepancies between the VLAN adhoc validation and yang model: https://github.com/sonic-net/sonic-buildimage/issues/12256

Would you be able to help address these?

Reply:
The original regex allows for Vlan0, Vlan00, Vlan000, Vlan01 ext...

Reply:
you're right, the first group in original regex was too broad. And thanks!